packages/rum/src/typesV2.ts

```diff
@@ -148,3 +148,15 @@ export type RumEventV2 =
   | RumViewEventV2 & ViewContextV2 & RumContextV2
   | RumLongTaskEventV2 & ActionContextV2 & ViewContextV2 & RumContextV2
   | RumActionEventV2 & ViewContextV2 & RumContextV2
+
+export interface User {
+  id?: number | string
+  email?: string
+  name?: string
+  [key: string]: unknown
+}
+
+export interface GlobalAttributes {
+  user: User
+  context: Context
```

Comment:
what about handling undefined values?
it could be a small payload optimization, avoiding to attach empty values to all events

Reply:
Ok for the `user`, but `context` would be a bit harder since it is expected to always be an object, no? (meaning: `DD_RUM.getRumGlobalContext()` is expected to return an object, not `undefined`)

Reply:
Could we have both behavior?
`DD_RUM.getRumGlobalContext()` returns an object but if empty context is not sent with the payload

Reply:
Ok! I'll change this during the assembly then.